src/ReasonReactRouter.rei

```diff
@@ -27,6 +27,9 @@ let unwatchUrl: watcherID => unit;
       for an example.
       */
 let dangerouslyGetInitialUrl: unit => url;
+/** expose url creation functions, usefull for SSR, create an url type from strings to give to serverUrl */
+let getUrl:
```

Comment:
Not sure `getUrl` makes sense when you're providing all the data. Why not `makeUrl`, `url`, or something similar?